firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceException.java

```diff
@@ -19,39 +19,38 @@
 
 /** The class for all Exceptions thrown by {@link FirebaseInstallationServiceClient}. */
 public class FirebaseInstallationServiceException extends FirebaseException {
-
-  public enum Code {
+  public enum Status {
     SERVER_ERROR,
 
     NETWORK_ERROR,
 
     UNAUTHORIZED
```

Comment:
what about INVALID_ARGUMENT, FORBIDDEN, NOT_FOUND, TOO_MANY_REQUEST?

Reply:
I will handle this in another PR.